doc/installing.rst

```diff
@@ -121,16 +121,15 @@ xarray itself is a pure Python package, but its dependencies are not. The
 easiest way to get everything installed is to use conda_. To install xarray
 with its recommended dependencies using the conda command line tool::
 
-    $ conda install xarray dask netCDF4 bottleneck
+    $ conda install -c conda-forge xarray dask netCDF4 bottleneck
 
 .. _conda: http://conda.io/
 
-We recommend using the community maintained `conda-forge <https://conda-forge.github.io/>`__ channel if you need difficult\-to\-build dependencies such as cartopy, pynio or PseudoNetCDF::
+If you require other `optional dependencies <#Optional dependencies>`_ add them to the line above.
```

Comment:
this link looks strange to me and I think it does not point anywhere. To fix it, I'd add a label in front of the `Optional dependencies` heading and then reference that:
```rst
`optional dependencies <optional-dependencies>`_
```
or
```rst
:doc:`optional-dependencies`
```